projects/1.18/assets/butchercraft/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+  "butchercraft.advancement.bonesaw.name": "骷髅的噩梦。",
+  "butchercraft.advancement.butcherknife.desc": "制作一把屠刀",
+  "butchercraft.advancement.butcherknife.name": "屠夫不杀五爪猪",
+  "butchercraft.advancement.cannibalism.desc": "试试去宰了一个人类。",
+  "butchercraft.advancement.cannibalism.name": "这里不是《环世界》……",
+  "butchercraft.advancement.cow.desc": "对一头牛使用屠刀（右击）",
+  "butchercraft.advancement.cow.name": "我牛肉呢？",
+  "butchercraft.advancement.everything.desc": "品尝所有肉类美食",
+  "butchercraft.advancement.everything.name": "贵者须食肉",
+  "butchercraft.advancement.everything_plus.desc": "品尝所有熟透的可食用部分。",
+  "butchercraft.advancement.everything_plus.name": "我全都要！",
+  "butchercraft.advancement.gutknife.desc": "制作一把穿肠刀",
+  "butchercraft.advancement.gutknife.name": "有胆量，小子。",
+  "butchercraft.advancement.heart.desc": "吃一颗生的心脏",
+  "butchercraft.advancement.heart.name": "贪心不足蛇吞象",
+  "butchercraft.advancement.hook.desc": "制作一个挂肉钩",
+  "butchercraft.advancement.hook.name": "如挂钩尖",
+  "butchercraft.advancement.pig.desc": "对一头猪使用屠刀（右击）",
+  "butchercraft.advancement.pig.name": "健康又美味",
+  "butchercraft.advancement.root.desc": "欢迎来到屠宰工艺！",
+  "butchercraft.advancement.root.name": "屠宰工艺",
+  "butchercraft.advancement.sheep.desc": "对一头羊使用屠刀（右击）",
+  "butchercraft.advancement.sheep.name": "香啊，很香啊",
+  "butchercraft.advancement.skinningknife.desc": "千钧一发",
+  "butchercraft.advancement.skinningknife.name": "制作一把剥皮刀",
+  "butchercraft.advancement.whole_cow.desc": "品尝每种熟牛肉",
+  "butchercraft.advancement.whole_cow.name": "废书焚去烹牛肉",
+  "butchercraft.advancement.whole_pig.desc": "品尝每种熟猪肉",
+  "butchercraft.advancement.whole_pig.name": "丰年留客足鸡豚",
+  "butchercraft.advancement.whole_sheep.desc": "品尝每种熟羊肉",
+  "butchercraft.advancement.whole_sheep.name": "沙晴草软羔羊肥",
+  "butchercraft.butcherknife.rightclick": "右击以屠宰",
+  "item.butchercraft.bbq_jar_item": "烧烤酱",
+  "item.butchercraft.bbq_ribs": "盘装炭烤排骨",
+  "item.butchercraft.beef_jerky": "牛肉干",
+  "item.butchercraft.beef_rib": "生牛排骨",
+  "item.butchercraft.beef_ribs": "生原切牛排骨",
+  "item.butchercraft.beef_roast": "生的原切牛扒",
+  "item.butchercraft.beef_scraps": "生牛肉丝",
+  "item.butchercraft.beef_stewmeat": "生牛碎肉",
+  "item.butchercraft.beef_wellington": "盘装惠灵顿牛排",
+  "item.butchercraft.bone_saw": "骨锯",
+  "item.butchercraft.brain": "生大脑",
+  "item.butchercraft.butcher_knife": "屠刀",
+  "item.butchercraft.butter": "黄油",
+  "item.butchercraft.cheese": "奶酪",
+  "item.butchercraft.chicken_fried_steak": "美式炸牛排",
+  "item.butchercraft.cooked_beef_rib": "熟牛排骨",
+  "item.butchercraft.cooked_beef_ribs": "熟原切牛排骨",
+  "item.butchercraft.cooked_beef_roast": "熟的原切牛扒",
+  "item.butchercraft.cooked_beef_scraps": "熟牛肉丝",
+  "item.butchercraft.cooked_beef_stewmeat": "熟牛碎肉",
+  "item.butchercraft.cooked_brain": "熟大脑",
+  "item.butchercraft.cooked_cubed_beef": "熟牛肉丁",
+  "item.butchercraft.cooked_cubed_lamb": "熟羊肉丁",
+  "item.butchercraft.cooked_cubed_pork": "熟猪肉丁",
+  "item.butchercraft.cooked_ground_beef": "熟牛绞肉",
```

```suggestion
  "item.butchercraft.cooked_ground_beef": "熟牛肉末",
```
虽然实际表现应该是从绞肉机里面弄出来的条状碎肉
只是因为读上去稍微有点怪所以提一下，不改也应该没问题